maps/upload.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<title>Planetary Response Network: file uploader</title>
+<link rel="stylesheet" type="text/css" href="/css/main.css">
+<main id="upload-page">
+  <form>
+    <div class="row">
+      <label>Files</label>
```

Comment:
This label isn't linked to a form input. It should either wrap the input, or be linked using `htmlFor` on the label and `id` on the input.